app-modules/task/src/Filament/Resources/TaskResource/Pages/ListTasks.php

```diff
@@ -137,6 +142,14 @@ public function setViewType(string $viewType): void
     protected function getHeaderActions(): array
     {
         return [
+            Action::make('test')
+                ->icon('heroicon-m-funnel')
+                ->iconButton()
+                ->badge(fn () => collect($this->table->getFilters())->map(fn (BaseFilter $filter) => $filter->getIndicators())->flatten()->count())
+                ->form(fn () => $this->table->getFiltersForm())
+                ->fillForm($this->tableFilters ?? [])
+                ->modalSubmitAction(false)
+                ->modalCancelAction(false),
```

Comment:
Did you intend to leave this test action in there?

Reply:
Ah, yes, this actually is the action. I have renamed it.